app/services/npq/create_or_update_profile.rb

```diff
@@ -22,6 +22,8 @@ def call
       participant_profile.school_urn = npq_validation_data.school_urn
       participant_profile.school_ukprn = npq_validation_data.school_ukprn
       participant_profile.save!
+
+      ParticipantProfileState.find_or_create_by!({ participant_profile: participant_profile })
```

Comment:
I'm assuming that the thinking is that we just want to create if it's a new profile?

Reply:
That's right.

Reply:
```suggestion
      ParticipantProfileState.find_or_create_by!(participant_profile: participant_profile)
```

nit: consistency